### PR TITLE
Fixes LCD - closes issue #116

### PR DESCRIPTION
**Pull request by SwindleA**
Fixed the number of rows in the display.

### CODE REVIEW

**Comment by ruthvikm** [comment]
LCD is not working with 64 bit OS. Use 32 bit raspberry pi OS to make it work

File: pi4micronaut-utils/src/main/java/com/opensourcewithslu/outputdevices/LCD1602Helper.java [hunk 1]
```diff
@@ -96,4 +102,94 @@ public void clearLine(int line)
...
     }
+
+    /**
+     * Turns off LCD Display. Works the same as backlight off, but a write or clear command will turn the display back on.
+     */
+    public void turnOff(){
+        lcdDisplay.off();
+    }
+
+    /// Below methods do not work
```
Comment (SwindleA) [inline pi4micronaut-utils/src/main/java/com/opensourcewithslu/outputdevices/LCD1602Helper.java:113]: I cannot get the methods involving the cursor and moving the display to work.

**Comment by ruthvikm** [comment]
This branch points to main, we need to merge these changes on to develop.

**Comment by SwindleA** [comment]
cannot get the cursor or display functions to work

**Review by ruthvikm — APPROVED** [review]
Looks good. I have added write text to position function

**Pull request merged**